src/ftdc_tools/rollups/client_perf.py

```diff
@@ -22,6 +22,14 @@ class Statistic:
     user_submitted: bool
 
 
+@dataclass
+class SummaryStatistic:
+    """A summary statistic."""
+
+    min: float
+    max: float
+
+
```

Comment:
Aren't all the statistics we're calculating summary statistics?

Reply:
It's Summary of summary Stats :D.

Removed it!